scripts/configure_dotenv.sh

```diff
@@ -22,7 +34,7 @@ cp "${DAPI_PATH}"/.env.example "${DAPI_PATH}"/.env
 # JS-SDK:
 FAUCET_ADDRESS=$(grep -m 1 "Address:" "${LOGS_PATH}"/mint.log | awk '{printf $3}')
 FAUCET_PRIVATE_KEY=$(grep -m 1 "Private key:" "${LOGS_PATH}"/mint.log | awk '{printf $4}')
-DPNS_CONTRACT_ID=$(yarn dashmate config get --config="${CONFIG}_1" platform.dpns.contract.id)
+DPNS_CONTRACT_ID=$(get_config platform.dpns.contract.id)
```

Comment:
This will be removed from dashmate. Please use hardcoded ID from contract crates

Reply:
added todo